L1Trigger/L1TMuonEndCap/interface/PtLUTVarCalc.h

```diff
@@ -1,22 +1,22 @@
-#ifndef L1TMuonEndCap_PtLutVarCalc_h
```

Comment:
No changes to this file except spaces ... will probably make the review easier if we just leave it untouched.